README.md

```diff
@@ -137,7 +138,87 @@ aws sso login --profile dpe-prod-admin
 aws eks update-kubeconfig --region us-east-1 --name dpe-k8 --role-arn arn:aws:iam::766808016710:role/eks_admin_role --profile dpe-prod-admin
 ```
 
-### Spacelift
+### Security and Audits in place for the EKS cluster
+AWS Guard is being used to perform audit trails for the EKS cluster, it involves 2
+components for the cluster:
+
+1. [EKS Audit Log Monitoring](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty-eks-audit-log-monitoring.html)
+2. [GuardDuty Runtime Monitoring](https://docs.aws.amazon.com/guardduty/latest/ug/runtime-monitoring-configuration.html)
+
+
+The initial configuration of these is handled through the `securitycentral` IT account.
+Runtime Monitoring is installed manually via terraform modules, allowing it to be torn
+down when we destroy the VPC and EKS cluster.
+
+
+In addition to this scanning that is in place we are also taking advantage of the

```

Comment:
Just a note, for trivy, lets focus on the HIGH/CRITICAL and issues that we can fix: https://github.com/Sage-Bionetworks/shiny-base/blob/e5770f31fe4a7d79150a25989a34ff3c3b1a78d1/.github/workflows/trivy.yml#L60-L69

Reply:
So Trivy is smart enough to know which vulnerabilities can and cannot be fixed if you configure it a certain way?

Reply:
I set https://github.com/Sage-Bionetworks-Workflows/eks-stack/pull/16/commits/50ac49a02bb0c5f4d379af87b276444581f6a563 `ignoreUnfixed: true`, 

We saw today that the number of issues flagged was around half